multimedia/VLC/Portfile

```diff
@@ -29,13 +29,6 @@ subport lib${name} {
     homepage            http://www.videolan.org/vlc/libvlc.html
 }
 
-set use_ffmpeg_VLC      no
-
-if {[tbool use_ffmpeg_VLC]} {
```

Comment:
There are two remaining `[tbool use_ffmpeg_VLC]` checks in the new [Portfile](https://github.com/RJVB/macfork-ports/blob/d6151a58a97a32ebb034f3dec82e619135ddd357/multimedia/VLC/Portfile), do you want to remove them?